xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -209,6 +209,17 @@
       "channels": { "type": "integer", "required": true }
     }
   },
+  "Player.Video.Stream": {
+    "type": "object",
+    "properties": {
+      "index": { "type": "integer", "minimum": 0, "required": true },
+      "name": { "type": "string", "required": true },
+      "language": { "type": "string", "required": true },
```

Comment:
Yes but does it always have one? If not the `language` property is not `required`.